Major: expand matrix URI documentation (#16502)
mdn/content: files/en-us/web/api/url/pathname/index.md

---
{{ApiRef("URL API")}}

> The **`pathname`** property of the URL interface is a [USVString](https://developer.mozilla.org/en-US/docs/Web/API/USVString) containing an initial `/` followed by the path of the URL, not including the query string or fragment (or the empty string if there is no path).

Reviewer: We are not using USVString in the documentation. Refer https://github.com/mdn/content/pull/15499. We've replaced such occurrences in the entire repository. 
```suggestion
The **`pathname`** property of the {{domxref("URL")}} interface is a string containing an initial `/` followed by the path of the URL, not including the query string or fragment (or the empty string if there is no path).
```

 
The path of the URL is a `/` delimited sequence of segments, which may each be one of `.`, `..` or a string with zero or more characters excluding `/`, `?`, and `#`.

> Some systems define the term *slug* to mean the final segment of a non-empty path if it identifies a page in human-readable keywords. For example, the URL https://example.org/articles/this-that-other-outre-collection has 'this-that-other-outre-collection' as its slug.

Reviewer 2: ```suggestion
Some systems define the term *slug* to mean the final segment of a non-empty path if it identifies a page in human-readable keywords. For example, the URL `https://example.org/articles/this-that-other-outre-collection` has `this-that-other-outre-collection` as its slug.
```

> Some systems use the `;` and `=` characters to delimit parameters and parameter values applicable to a path segment. For example, with the URL https://example.org/users;id=42/tasks;state=open?sort=modified, a system might extract and use the path segment parameters `id=42` and `state=open` from the path segments `users;id=42` and `tasks;state=open`.

Reviewer 2: ```suggestion
Some systems use the `;` and `=` characters to delimit parameters and parameter values applicable to a path segment. For example, with the URL` https://example.org/users;id=42/tasks;state=open?sort=modified`, a system might extract and use the path segment parameters `id=42` and `state=open` from the path segments `users;id=42` and `tasks;state=open`.
```
